core/common.h

```diff
@@ -1,7 +1,7 @@
 /* This header file contains general (not BESS specific) C definitions */
 
-#ifndef _COMMON_H_
-#define _COMMON_H_
+#ifndef CORE_COMMON_H_
```

Comment:
Sorry for the nitpick -- you might prepend BESS_ to all of these, to match the header guard style from the style guide.